### ISSUE TITLE
Problem with types

### ISSUE THREAD

**Issue by Morb0** [RealSpeaker/telegraf-session-local#117]
Hey, just install package and got types error:
```

node_modules/telegraf-session-local/lib/session.d.ts:34:3 - error TS2309: An export assignment cannot be used in a module with other exported elements.

34   export = LocalSession
     ~~~~~~~~~~~~~~~~~~~~~

```

Seems it because also interface exported. I think change `export = LocalSession` to `export default LocalSession` can fix this.

**Comment by EdJoPaTo** [comment]
I use this as it is in multiple bots. Check for example [my template](https://github.com/EdJoPaTo/telegram-typescript-bot-template/blob/42dc65a01581d78ed92c3cba91ce53137f23e8fe/source/bot/index.ts#L6) which also works.

Typings are not ideal but they work good enough for me. The only thing that would really improve types in my opinion would be a (breaking) refactoring into native typescript. Then types are exactly the same as the underlying JavaScript.

**Comment by Morb0** [comment]
I tried the import like you, but still the same problem I use the latest version which is 2.1.0 and my telegraf version is 4.0.3.

**Comment by EdJoPaTo** [comment]
If you have `esModuleInterop` enabled (in `tsconfig.json`) you might need an import without `* as`. Personally I try to avoid `esModuleInterop` as it also works without (as long as the modules I use dont have it).

**Comment by Morb0** [comment]
Yes, I do the same. I have already tried with both enabled and disabled `esModuleInterop`, but this is an error in the export types, WebStorm highlight this.

**Comment by TEHbKA-dev** [comment]
Hey!
How to solve this problem?

> An export assignment cannot be used in a module with other exported elements.
> 
> 34   export = LocalSession

**Comment by gabrieljaegerde** [comment]
Still having this issue too! not typescript compatible

**Comment by romap0** [comment]
Using `"skipLibCheck": true` in tsconfig.json fixed this for me

**Comment by EdJoPaTo** [comment]
As people check this issue I will suggest it here again: I strongly suggest using [grammY](https://grammy.dev/) over Telegraf as it has a lot of benefits while being mostly API compatible. Its documentation is way better and was built with the learnings from Telegraf which allows for better performance, simpler and more powerful code syntax.

If you care for TypeScript (which is probably true as you seek this issue) then grammY will be way more helpful and correct than Telegraf can as grammY was built with types in mind.
